docs/guides/integrating-optimize.md

```diff
@@ -0,0 +1,118 @@
+2. Click the **Applications** tab.
+3. Select the **Optimize** box at the bottom of the page to launch Optimize.
+
+You can begin analyzing reports and dashboards with just two process versions. However, the more process versions you work with in Optimize, the more performance attributes and data trends you’ll be able to study. For the purposes of this guide, we’ve preconfigured several processes to demonstrate Optimize’s capabilities.
+
+## Create and analyze dashboards
+
+Within Optimize, **reports** are based on a *single* visualization, similar to a single chart or graph. **Dashboards** are aggregations of these visualizations, similar to a full spreadsheet of data collections, or a combination of several comparative charts and graphs. **Collections** are groups of these data sets, similar to project folders for organizational purposes where we can nest a series of dashboards and/or reports within.
+
+Once you open Optimize, you’ll first view the homepage for these collections, dashboards, and reports.
+
+To create a collection on the **Home** page, select **Create New > New Collection**. Name your collection and click **Create Collection**.
+
+Next, click on the collection to view its contents. You’ll notice you can again select Create New to draft new reports and dashboards.
+
+Let’s create a dashboard inside our first collection. Take the following steps:
+
+1. Return to the **Home** page to review your collections, dashboards, and reports. You’ll be able to view all process instances you’ve already run and retrieve additional data on these instances within the Camunda engine.
+2. Select the collection where you’d like to create a dashboard.
+3. Click **Create New > New Dashboard**.
+4. Optimize offers preconfigured dashboard templates, or you can start from a blank dashboard. In this example, we’ll select a preconfigured template by clicking the **Process performance overview** option.
+5. Under **Select Process**, choose the process you’d like to analyze and the version.
+6. Click **Create Dashboard**.
+7. Name your dashboard, and add any preferred reports or filters. Click **Save**.
+
+**Image placeholder**
+
+In the sample above, Optimize drafted a dashboard filled with reports for review. These reports include objectives like process instance counts, aggregated process duration, active incidents, and heatmaps. 
+
+Select **Edit > Add a Report** to incorporate additional reports you’ve already created (see #create-and-access-reports below). Click and drag the reports on the grid to arrange the dashboard to your liking.
+
+Share your dashboard by clicking the **Share** tab. Toggle to **Enable sharing**, and copy or embed the provided link. Colleagues without access to Optimize can still view your report with the shared link.
+
+## Create and access reports
+
+To create a custom report based on a key performance indicator (KPI) you’d like to analyze, and to incorporate this report into a dashboard, follow the steps below:
```

Comment:
YES! Linking this back to the KPI is key for the business users. I'm interested in creating an article about which KPIs can be tracked and how

Reply:
A doc on these concrete examples/KPIs would be great -- filing way for future doc writing opportunities!